scripts/setup_environment.ts

```diff
@@ -44,13 +44,15 @@ module.exports = async function (callback: Truffle.ScriptCallback) {
 
     // Create balance and approval for tokens
     for (let account = 0; account < argv.numAccounts; account++) {
-      for (let token = 0; token < argv.numTokens; token++) {
-        if (token == 0) {
-          await owlProxy.mintOWL(accounts[account], amount);
-        } else {
-          await tokens[token].contract.methods.mint(accounts[account], amount);
-        }
-        await tokens[token].approve(instance.address, amount, {
+      // Mint and approve OWL
+      await owlProxy.mintOWL(accounts[account], amount);
```

Comment:
tricky.... Its a different method!

Reply:
It is. I am now going to use my new found understanding here to deal with #822